infrastructures/kubernetes/pom.xml

```diff
@@ -69,6 +69,10 @@
             <groupId>com.squareup.okhttp3</groupId>
             <artifactId>okhttp</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okio</groupId>
```

Comment:
do we really need this new dependency?
Are licenses compatible?

Reply:
We have a policy in our poms that requires to explicitly declare any dependency the code uses. `okio` is a transitive dep of `okhttp` and I happen to use one of the classes in the codebase now. So this is not a new dependency, it's just an explicitly re-declared transitive one.